apps/buildog/src/app/(auth)/layout.tsx

```diff
@@ -1,13 +1,12 @@
-import { Metadata } from "next";
+"use client";
```

Comment:
We don't use hooks here. We don't need to describe this page is a client page. We can remove this line